lib/packwerk/file_processor.rb

```diff
@@ -32,8 +32,8 @@ def call(file_path)
 
       result = []
       if node
-        @node_processor = @run_context.node_processor_for(filename: file_path, ast_node: node)
```

Comment:
It’s interesting that we had an unnecessary instance variable here. Looking at the history, it was originally used by a `#visit` method; that method got removed when we extracted `Packwerk::NodeVisitor` but the instance variable was left behind.